### PR TITLE
chore: Fix typos and improve grammar of README.md

### PR DESCRIPTION
**Pull request by gmfrasca**
## The issue resolved by this Pull Request:
Resolves #annoyance

## Description of your changes:
Add context and improve grammar of prereqs and installation instructions in README

## Testing instructions
RTD

## Checklist
- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -604,7 +608,7 @@ The specific tests that are executed when you run `make test` can include unit t
 
 **To deploy DSPO as a developer :**
 
-Follow the instructions from [here](#deploy-the-operator-standalone) to deploy the operator standalone.
+Follow the instructions from [here](#deploy-the-standalone-operator) to deploy the operator standalone.
```
Comment (franciscojavierarceo) [inline README.md:611]:
```suggestion
Follow the instructions from [here](#deploy-the-standalone-operator) to deploy the standalone operator.
```

**Review by DharmitD — APPROVED** [review]
/Approve

**Review by franciscojavierarceo — APPROVED** [review]

**Pull request merged**